ntoskrnl/inbv/inbv.c

```diff
@@ -1040,13 +1057,21 @@ DisplayBootBitmap(IN BOOLEAN TextMode)
             InbvBufferToScreenBlt(Buffer, 0, 0, 22, 9, 24);
         }
 
-        LineBmp = InbvGetResourceAddress(IDB_ROTATING_LINE);
-        if (LineBmp && TempRotBarSelection == RB_PROGRESS_BAR)
+        /*
+         * Add a rotating bottom horizontal bar when using a progress bar,
+         * to show that ReactOS can be still alive when the bar does not
+         * appear to progress.
+         */
+        if (TempRotBarSelection == RB_PROGRESS_BAR)
```

Comment:
I do this in order just load the rotating line resource *only* if the RotBar selection is PROGRESS_BAR.

Reply:
Agree.